schemas/v0.1/sample.yml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=./yaml.schema.json
+
+version: v0.1
+
+workingDir: ai-project/	# default artifacts folder
+
+environmentFolder: ai-project/env/	# .env files location
+
+dataset:        # dataset feature definition
+  content:      # recognizable dataset files
+    - dataset/output/*.json 	# glob pattern
```

Comment:
No need to restrict the file extension for dataset? More dataset formats would be supported in future, including JSONL, CSV, Parquet...

Reply:
Forgot it's just a dev sample, ignore this comment.

Reply:
No need in schema, since the pattern could also be something like `output/*` or `**/json/*`. It's totally cusotmizable.
But ok to check file format on runtime.